frontend/src/lib/components/SeriesGlyph.tsx

```diff
@@ -36,7 +36,9 @@ export function SeriesLetter({ className, hasBreakdown, seriesIndex, seriesColor
                           color: color,
                           backgroundColor: hexToRGBA(color, 0.15),
                       }
-                    : undefined
+                    : {
+                          color: 'var(--default)',
```

Comment:
I added this for the case where there's a breakdown. I think since `<SeriesLetter />` was inside of a `<LemonRow />` component, a `color: var(--muted)` was being applied from one of the LemonRow css rules. This resulted in the letter being muted instead of black. Feels a bit hacky so if this needs to change lemme know!